src/Admin/PageAdmin.php

```diff
@@ -266,8 +266,8 @@ protected function configureDatagridFilters(DatagridMapper $filter)
                 'field_options' => [
                     'required' => false,
                     'choices' => [
-                        'hybrid' => $this->trans('hybrid'),
-                        'cms' => $this->trans('cms'),
+                        'hybrid' => $this->getTranslator()->trans('hybrid', [], 'SonataPageBundle'),
```

Comment:
wouldn't it just be enough to remove the 'choice_translation_domain' part?

then the translator would translate it anyway?

Reply:
Indeed, should be better knwo